mmdeploy/codebase/mmdet3d/deploy/voxel_detection.py

```diff
@@ -50,107 +76,141 @@ def build_pytorch_model(self,
             model_checkpoint (str): The checkpoint file of torch model,
                 defaults to `None`.
             cfg_options (dict): Optional config key-pair parameters.
+
```

Comment:
could remove this func and use that from father class.

Reply:
done.